### PR TITLE
count users via notify

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Looks good! 
* But please remove all special users and show groups only (count users + tabs per group).
* Please use this wording: "Active users: x (y connections)"
* "Active users" show always the same value as "Active users tabs"

**Comment by FinnStutzenstein** [comment]
@jsaalfeld please discuss with @emanuelschuetze

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze I did changed a few things. Please check again

File: client/src/app/core/services/count-users.service.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+const REQUEST_NAME = 'count-user-request';
+const RESPONSE_NAME = 'count-user-response';
+
+/**
+ */
+@Injectable({
+    providedIn: 'root'
+})
+export class CountUsersService extends OpenSlidesComponent {
```
Comment (tsiegleauq) [inline client/src/app/core/services/count-users.service.ts]: Documentation is missing completely

File: client/src/app/core/services/filter-list.service.spec.ts [hunk 2]
```diff
@@ -1,18 +0,0 @@
-import { TestBed } from '@angular/core/testing';
-
-import { FilterListService } from './filter-list.service';
-import { E2EImportsModule } from '../../../e2e-imports.module';
-
-describe('FilterListService', () => {
```
Comment (tsiegleauq) [inline client/src/app/core/services/filter-list.service.spec.ts]: Should have been renamed, not deleted

File: client/src/app/core/services/sort-list.service.spec.ts [hunk 3]
```diff
@@ -1,18 +0,0 @@
-import { TestBed } from '@angular/core/testing';
-
-import { SortListService } from './sort-list.service';
-import { E2EImportsModule } from '../../../e2e-imports.module';
-
-describe('SortListService', () => {
```
Comment (tsiegleauq) [inline client/src/app/core/services/sort-list.service.spec.ts]: Should have been renamed, not deleted

File: client/src/app/site/common/components/legal-notice/legal-notice.component.ts [hunk 4]
```diff
@@ -1,12 +1,50 @@
...
 export class LegalNoticeComponent implements OnInit {
-    public constructor() {}
+    public token: string = null;
+    public stats: CountUserStatistics = null;
+
+    public constructor(private countUsersStatisticService: CountUsersStatisticsService) {}
 
     public ngOnInit(): void {}
+
+    public countUsers(): void {
```
Comment (tsiegleauq) [inline client/src/app/site/common/components/legal-notice/legal-notice.component.ts]: should get an own page if you want to leave that code

Comment (FinnStutzenstein) [reply client/src/app/site/common/components/legal-notice/legal-notice.component.ts]: ok

**Comment by FinnStutzenstein** [comment]
@tsiegleauq Ping.

**Comment by emanuelschuetze** [comment]
Rebased

**Pull request merged**